pages/faqs.md

```diff
@@ -17,25 +17,25 @@ To learn more about what it means for the devil to be in the details, check out
 [Overkill](http://www.kytrinyx.com/talks/overkill) and
 [Succession](http://www.kytrinyx.com/talks/succession) talks by Katrina.
 
-### I get the error "Sorry, we could not authenticate you from GitHub" when trying to log in. What should I do?
-
-This means that GitHub isn't willing to verify who you are. That can be because you choose to not give permission or it might be because your GitHub account is not properly configured. A common problem is that you haven't verified your email address on GitHub. You can check that in your [GitHub email settings](https://github.com/settings/emails).
+## The Basics
 
-### How do I delete my account?
+### I'm lost. Where do I go?
+You're in luck. Here's a [step-by-step guide](https://teams.exercism.io/getting-started) to get you started.
 
-You can delete your account by following the link at the bottom of your [settings page](https://exercism.io/my/settings).
+If something is still really unclear for you or not working then it might be the same for others so we'd appreciate your suggestions. Refer to [Opening an Issue](#opening-an-issue) below for instructions on how you can help us help others.

```

Comment:
```suggestion
If something is still unclear for you or not working then it might be the same for others so we'd appreciate you letting us know. Refer to [Opening an Issue](#opening-an-issue) below for instructions on how you can help us help others.

```